app/models/project.rb

```diff
@@ -32,12 +32,7 @@ module Types
   validate :project_with_school_id_has_school_project
   validate :school_project_school_matches_project_school
 
-  default_scope -> { where.not(project_type: Types::SCRATCH) }
-
   scope :internal_projects, -> { where(user_id: nil) }
-  scope :only_scratch, lambda { |only_scratch|
-    only_scratch ? unscoped.where(project_type: Project::Types::SCRATCH) : self
-  }
```

Comment:
I thought the `only_scratch` stuff was still needed on the ExCS side? No worries if I misunderstood or that's changed

Reply:
Great 👍